src/det.jl

```diff
@@ -44,6 +44,13 @@ end
 
 det(F::LU) = det(F.U) * _parity(F.p)
 
+function logabsdet(A::Union{LowerTriangular{<:Any,<:StaticMatrix},
+                            UpperTriangular{<:Any,<:StaticMatrix}})
+    checksquare(A)
```

Comment:
My recollection from years ago (yikes!) is that this method is slow. Have you checked if it is ellided? We can do it statically in a variety of ways, the simplest of which would be to add type parameter constraints like `<:StaticMatrix{S,S} where S`

Reply:
According to `@code_warntype`, it is fully elided.

In any case, it is used in multiple places in the package.